PagerDutyEvent.cfc

```diff
@@ -0,0 +1,125 @@
+component accessors = "true" {
+
+	property name = "component" type = "string";
+	property name = "customDetails" type = "struct";
+	property name = "dedupKey" type = "string";
```

Comment:
Not sure if that "abbreviation" fits the coding standards? In either event, shouldn't it be "dedupeKey"?

Reply:
Huh. I guess that's what PagerDuty calls it, so ... never mind.  (And :rageguy: at Pager Duty for "dedup")

Reply:
yeah i have a hard time reconciling what the field is called externally for consistency w/ their API, vs. aligning it w/ our convention. i'll build out the README.md a bit more, since i'm makin' a change anyway